docs/docs/debugging.md

```diff
@@ -1,8 +1,8 @@
 ---
 title: Debugging
+overview: true
 ---
 
-This is a stub. Help our community expand it.
+While building your site, you’ll most likely encounter some bugs. This guide helps you set up debugging for various Gatsby components to help you spot and squash bugs more easily.
```

Comment:
How about "you'll sometimes encounter bugs". That's more optimistic ;-)

It should be "these guides".

I'd use a different word than "components" as that'd be confusing in the context of React.